mu-plugins/blocks/sidebar-container/postcss/style.pcss

```diff
@@ -44,3 +44,8 @@
 		scroll-margin-top: var(--wp-local-header-offset, 0);
 	}
 }
+
+/* Set up the custom properties. These can be overridden by settings in theme.json. */
+:where(body) {
+	--wp--custom--wporg-sidebar-container--spacing--margin--top: var(--wp--preset--spacing--edge-space);
```

Comment:
Default value set to the existing value so that the Documentation site doesn't require any changes